kvdb-rocksdb/src/lib.rs

```diff
@@ -232,8 +232,8 @@ struct DBAndColumns {
 	column_names: Vec<String>,
 }
 
-fn static_property_or_warn(db: &DB, prop: &str) -> usize {
-	match db.property_int_value(prop) {
+fn static_property_or_warn(db: &DB, col: &ColumnFamily, prop: &str) -> usize {
```

Comment:
can we make it a method on `DBAndColumns`, something like `fn static_property_or_warn(&self, column: u32, property: &str) -> usize`?